packages/teams-js/src/internal/telemetry.ts

```diff
@@ -50,7 +50,7 @@ export enum ApiVersionNumber {
   V_3 = 'v3',
```

Comment:
Will making ApiVersionNumber a const also help? (if it can be made as a const)

Reply:
Maybe one other question I will attach here: is there something like a linting rule we could enable to prevent people from making enums that don't use the const keyword?

Reply:
Technically yes, making any enum a const enum that can be should theoretically reduce package size, but since it's such a small enum it would be negligible change. (I think it would saves 12 bytes). The reason making the `ApiName` telemetry strings `const` saves so much space is because it is such a big enum with both a lot of values, and the left side names of the values are a lot of characters as well. In typescript, enums by default have reverse mappings, but const enums do not, so making it const allows all of the left side names to be symbolized in the minified code.

Reply:
> Maybe one other question I will attach here: is there something like a linting rule we could enable to prevent people from making enums that don't use the const keyword?

We could definitely make a lint rule, however keep in mind that there are times when a regular, non-const enum is the right move, so if we did make a rule, it should just be a warning/reminder rather than a blocking error.